hvac/api/auth_methods/aws.py

```diff
@@ -26,6 +26,7 @@ def configure(
         endpoint=None,
         iam_endpoint=None,
         sts_endpoint=None,
+        sts_region=None,
```

Comment:
We'll need to move this to the end of the parameter list to avoid breaking users setting their arguments positionally.